src/ontology/cl-edit.owl

```diff
@@ -29245,13 +29247,13 @@ AnnotationAssertion(oboInOwl:inSubset obo:CL_4023018 cl:BDS_subset)
 AnnotationAssertion(rdfs:label obo:CL_4023018 "pvalb GABAergic cortical interneuron")
 EquivalentClasses(obo:CL_4023018 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000013502)))
 
-# Class: obo:CL_4023019 (L5/6 cck, vip GABAergic interneuron (Mmus))
+# Class: obo:CL_4023019 (L5/6 cck, VIP GABAergic interneuron (Mmus))
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33184512") obo:IAO_0000115 obo:CL_4023019 "A vip GABAergic cortical interneuron that expresses cck. L5/6 cck cells have soma found mainly in L5 and L6 and have large axonal arborization.")
 AnnotationAssertion(terms:contributor obo:CL_4023019 <https://orcid.org/0000-0001-7258-9596>)
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023019 "L5/6 CCK Vip (Mus musculus)")
```

Comment:
Vip --> VIP

Reply:
Weird I thought I changed all of these. anyways I will have a look with this next commit